native_mate/arguments.cc

```diff
@@ -55,7 +50,7 @@ v8::Local<v8::Value> Arguments::ThrowError() const {
 
   return ThrowTypeError(base::StringPrintf(
       "Error processing argument at index %d, conversion failure from %s",
-      next_, V8TypeAsString((*info_)[next_]).c_str()));
+      next_, V8TypeAsString(isolate_, (*info_)[next_]).c_str()));
```

Comment:
@zcbenz The `isolate_` won't ever be `NULL` right?

It looked like it wouldn't be since a few lines before this `ThrowTypeError` calls `MATE_THROW_EXCEPTION` which does an unchecked `isolate->ThrowException(value)`